File: src/main/java/com/GoldenMine/parser/postposition/IPostPosition부사어에.java
package com.GoldenMine.parser.postposition;

public class IPostPosition부사어에 implements IPostPosition {
    @Override
    public String getType() {
        return "부사어에";
    }

    @Override
    public int verify(CharSequence last) {
        if(last.length() >= 1) {
            char ch = last.charAt(last.length() - 1);
            //System.out.println("V: " + ch);
            return (ch == '에') ? 1 : -1;
        } else {
            return -1;
        }
    }

    @Override
    public JosaCommunity getJosaCommunity() {
        return JosaCommunity.격조사;
    }
}

File: src/main/java/kr/goldenmine/parser/postposition/IPostPosition부사어에.kt
package kr.goldenmine.parser.postposition

class IPostPosition부사어에 : IPostPosition {
    override val type: String
        get() = "부사어에"

    override val josaCommunity: JosaCommunity
        get() = JosaCommunity.격조사

    override fun verify(last: CharSequence): Int {
        if (last.length >= 1) {
            val ch = last[last.length - 1]
            //System.out.println("V: " + ch);
            return if (ch == '에') 1 else -1
        } else {
            return -1
        }
    }
}
